### DISCUSSION TITLE
Where is the implement of Lifecycle in IOS?

### DISCUSSION THREAD

**Discussion by king-ma1993** [arkivanov/Essenty#87] [Q&A]
Where is the implement of Lifecycle in IOS?

**Comment by arkivanov** [comment]
There is no any special implementation for iOS. You can use `LifecycleRegitry` and control the lifecycle manually. Check out examples:

1. https://github.com/JetBrains/compose-jb/blob/aa64d2d232d7f8fd310f7423fec9a3f8dd90a840/examples/todoapp/ios/ios/ContentView.swift#L17
2. https://github.com/arkivanov/Decompose/blob/21dc79db9aed2680cf317c74597714a8be79a17c/sample/app-ios/app-ios/app_iosApp.swift#L21

Also please see readme - https://github.com/arkivanov/Essenty#content
